CefSharp.Example/RenderProcessMessageHandler.cs

```diff
@@ -17,9 +17,12 @@ void IRenderProcessMessageHandler.OnFocusedNodeChanged(IWebBrowser browserContro
 
         void IRenderProcessMessageHandler.OnContextCreated(IWebBrowser browserControl, IBrowser browser, IFrame frame)
         {
-            const string script = "document.addEventListener('DOMContentLoaded', function(){ alert('DomLoaded'); });";
+            if (frame.IsMain)
```

Comment:
I think we should probably add a comment here regarding multiple calls to this method, as not everyone reads the xml doc.